### ISSUE TITLE
Incomplete install: npm install air-ng

### ISSUE THREAD

**Issue by xErik** [unwitting/air-ng#6]
The command `npm install air-ng` does not install any sources.

**Comment by unwitting** [comment]
Hi @xErik - all sorted out?

**Comment by xErik** [comment]
Well, I saw that the README speaks about a future state.

I tried the above command in Kali as well as on OSX, each time the `src/` folder is missing.

I assume, you simply did not publish the current version completely on NPM? :-)

**Comment by unwitting** [comment]
It's published on npm - the `src/` dir is deliberately omitted from the npm-published module though, since the ES5-compiled `lib/` is listed as the package's main entry point. The ES6 `src` is what it's developed in, but it's publised as ES5 since that way no-one needs to run it with `babel-node` or the like :)

**Comment by unwitting** [comment]
If the `lib/` directory is missing, then that's a problem! But once you've done `npm install air-ng` are you able to do the following in node, and see the exports? If not, let me know :)

```js
var air = require('air-ng');
console.log(air);
```

**Comment by xErik** [comment]
Nope, it is not there.

```console
$ npm install air-ng
air-ng@0.0.2 node_modules/air-ng
$ l node_modules/air-ng/
total 24
drwxr-xr-x  5 me  staff   170 Aug 23 19:57 .
drwxr-xr-x  3 me  staff   102 Aug 23 19:57 ..
-rw-r--r--  1 me  staff    15 Aug 18 12:30 .npmignore
-rw-r--r--  1 me  staff  1676 Aug 18 13:17 README.md
-rw-r--r--  1 me  staff  1387 Aug 23 19:57 package.json
```

**Comment by unwitting** [comment]
Interesting! I'll sort that out soon, I pretty much only published to npm
to reserve the name until v1.

**Comment by xErik** [comment]
Does NPM ignore files and folders listed in `.gitignore`?

*.gitignore*
```
+node_modules/
+lib/
```

**Comment by unwitting** [comment]
Nope it doesn't, I don't know how that happened. Fixed by 0.0.3 :)

**Comment by unwitting** [comment]
Cheers for raising - if you're interested in using the project, at the moment you have access to air.interfaces, which exposes the ability to take down, bring up, and change mode of interfaces using ifconfig and iwconfig, either directly or in nice wrappers like `air.interfaces.down().then...`.
